Tools/Scripts/libraries/webkitcorepy/webkitcorepy/terminal.py

```diff
@@ -45,10 +45,15 @@ def choose(cls, prompt, options=None, default=None, strict=False, numbered=False
         response = None
         while response is None:
             if numbered:
-                numbered_options = ['{}) {}'.format(i + 1, options[i]) for i in range(len(options))]
+                numbered_options = ['{}) {}'.format(
```

Comment:
nit: can this be on the same line as the one below? Or making these into variables on their own line to make them easier to read?